packages/@webex/plugin-meetings/src/hashTree/hashTreeParser.ts

```diff
@@ -958,10 +958,6 @@ class HashTreeParser {
       );
 
       dataSet.timer = setTimeout(async () => {
-        LoggerProxy.logger.info(
```

Comment:
removed that log, just because we have quite a lot of logs already in this area: one when setting the timer and another one when comparing the hashes once timer expires, so this one is redundant